packages/nimble-components/src/patterns/anchor/tests/anchor-patterns.mdx

```diff
@@ -0,0 +1,22 @@
+import { Controls, Canvas, Meta, Title } from '@storybook/blocks';
+import * as anchorPatternStories from './anchor-patterns.stories';
+
+<Meta of={anchorPatternStories} />
+<Title of={anchorPatternStories} />
+
+Anchor components in nimble should behave like native anchors in a number of different ways that are difficult to unit test in an automated manner.
+Therefore, this story contains a native anchor element along with all of the nimble components that are anchors or contain anchors.
+The following behaviors should be true of the nimble components:
+
```

Comment:
Some of these don't work today in some browsers. I didn't test exhaustively but noticed:
1. focusing via tab doesn't show URL preview in Safari. That matches the behavior of native links.
2. pressing Space on a focused link doesn't navigate the link in Safari (but Enter does). That matches the behavior of the native link.
    - the anchor tab is the exception. It does navigate on Space.
4. no visible link on drag in macOS Firefox (as we [discussed](https://github.com/ni/nimble/pull/1910#discussion_r1519884688) in the drag PR)

Thoughts on doing a more comprehensive audit and/or tracking the gaps? We could add notes in this list or file issues for the gaps and link them from here.

Reply:
I don't have a strong opinion, but I'd lean towards tracking those browser differences here. As you mentioned with the tab preview in safari, not all the differences are actually bugs since nimble behavior matches native behavior.

If there are specific bugs that we think we can address in nimble, then we can file issues for those. Otherwise, I think we'd just end up with a bunch of bugs documenting browser differences, which I'm not sure we want.

Reply:
Good point about filing bugs for browser differences; I'm good with documenting the differences here.

Reply:
I've documented the differences here as sub-bullets of a given behavior.